spec/models/corporate_information_page_spec.rb

```diff
@@ -17,4 +17,64 @@
       expect(described_class.new(content_store_response).default_organisation).to be_nil
     end
   end
+
+  describe "#corporate_information?" do
+    it "presents corporate information groups on about pages" do
+      expect(described_class.new(content_store_response).corporate_information?).to be(true)
+    end
+
+    it "does not present corporate information groups on about pages if information isn't available" do
+      content_store_response["details"]["corporate_information_groups"] = nil
+
+      expect(described_class.new(content_store_response).corporate_information?).to be(false)
+    end
+  end
+
+  describe "#corporate_information" do
+    subject(:corporate_information) { described_class.new(content_store_response).corporate_information }
+
+    it "includes group headings" do
+      expect(corporate_information.first[:title]).to eq("Access our information")
+      expect(corporate_information.first[:id]).to eq("access-our-information")
+    end
+
+    context "with 'Access our information' section" do
+      it "includes the organisation chart as the first group link if it is available" do
+        expect(corporate_information.first[:links].first).to eq({ title: "Our organisation chart", path: "https://data.gov.uk/dataset/04427362-663e-49e0-9103-8bc01dcaa2c7/organogram-of-staff-roles-and-salaries" })
+      end
+
+      it "includes group links and titles that have matching guids" do
+        expect(corporate_information.first[:links].last).to eq({ title: "Statistics at DHSC", path: "/government/organisations/department-of-health-and-social-care/about/statistics" })
+      end
+
+      it "excludes group links that do not have matching guids" do
+        groups = content_store_response.dig("details", "corporate_information_groups")
+        groups.first["contents"] << "123"
+
+        corporate_information = described_class.new(content_store_response).corporate_information
+
+        expect(corporate_information.first[:links].count).not_to eq(groups.count)
```

Comment:
Non-blocking note: You could make this `.to eq(groups.count - 1)`, I think? And it would be a bit of a tighter test, because it wouldn't pass if all the links were gone, or there were a thousand additional links? (I'm assuming there's more than one link, of course, so that setting the bogus GUID doesn't take this from 1 link to 0)